spec/features/articles/user_edits_article_spec.rb

```diff
@@ -1,37 +1,32 @@
 require 'rails_helper'
 
 feature 'User edits an article', %q(
-	As an authenticated user 
-	I want to update an item's information 
-	So that I can correct errors or provide new information
+  As an authenticated user
+  I want to update an item's information
+  So that I can correct errors or provide new information
 
-	Acceptance criteria:
+  Acceptance criteria:
 
-	[x] The user must be logged in
-	[x] The user can edit an article
-	[x] Only the user who posted the article can edit it
+  [x] The user must be logged in
+  [x] The user can edit an article
+  [x] Only the user who posted the article can edit it
 ) do
+  let!(:user) { FactoryGirl.create(:user) }
+  let!(:article) { FactoryGirl.create(:article, user: user) }
 
-let!(:user) { FactoryGirl.create(:user) }
-let!(:article) { FactoryGirl.create(:article, user: user) }    
+  scenario 'User edits an article' do
+    sign_in_as(user)
+    visit article_path(article.id)
 
-scenario 'User edits an article' do
+    click_on 'Edit'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.